ui/desktop/app/components/settings-card/server/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+
+export default class SettingsCardServerComponent extends Component {
+  // =services
+  @service intl;
+
+  // =attributes
+  /**
+   * Returns the provider name for HCP and generic name for non HCP servers
+   * @type {string}
+   */
+  get providerName() {
+    const regex = /hashicorp\.cloud/;
+    if (this.args.model.serverInformation?.match(regex)) {
```

Comment:
Do we anticipate having to know if is HCP or not somewhere else?

Asking just in case we want to apply the pattern we have in some models I.e: add a function isHcp where it receives a value, perhaps serverUrl, and returns true or false if belongs to HCP.

This way we can re-use the logic that determinate if is HCP.

Reply:
hmm, I don't know if this belongs in the model since this seems like a parsing that the UI needs for this specific use case.

Reply:
yep. I was wondering in case we have more cases within the DC where we need to determinate if the controller is hcp or not hcp we could do something more generic.
But seems at the moment, this is the only case we have, so all good KISS 😄